app/javascript/components/Header/Header.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function Header() {
+  return (
+    <div className="jumbotron">
```

Comment:
Use BEM

Reply:
@shem8 No need for BEM if we use CSS modules (preferred).